fundingmanager.go

```diff
@@ -729,17 +740,18 @@ func (f *fundingManager) handleFundingSignComplete(fmsg *fundingSignCompleteMsg)
 		openChan, confHeight, confBlockIndex := resCtx.reservation.DispatchChan()
 		// This reservation is no longer pending as the funding
 		// transaction has been fully confirmed.
-		f.deleteReservationCtx(peerID, chanID)
+		f.deleteReservationCtx(peerKey, chanID)
 
 		fndgLog.Infof("ChannelPoint(%v) with peerID(%v) is now active",
-			fundingPoint, peerID)
+			fundingPoint, peerKey)
 
 		// Now that the channel is open, we need to notify a number of
 		// parties of this event.
 
 		// First we send the newly opened channel to the source server
 		// peer.
-		fmsg.peer.newChannels <- openChan
+		peer := f.findPeer(peerKey)
```

Comment:
`findPeer` should return an error in the case that the peer can't be found. Otherwise, lines like the one below could run into a panic if the remote peer disconnect mid-way during the funding process.